charts/xui-webapp/values.preview.template.yaml

```diff
@@ -16,7 +16,7 @@ nodejs:
      HEALTH_WORK_ALLOCATION_TASK_API: http://wa-task-management-api-aat.service.core-compute-aat.internal/health
      HEALTH_ROLE_ASSIGNMENT_API: http://am-role-assignment-service-aat.service.core-compute-aat.internal/health
      HEALTH_CASE_CASEWORKER_API: http://rd-caseworker-ref-api-aat.service.core-compute-aat.internal/health
-     STUB: true
+     STUB: false
```

Comment:
_Question:_
Is this deliberate?

Reply:
Yes this for the preview env, change this will enable the real api call in preview env, previously it's running on mock.